### PR TITLE
add offboarding template

### PR DESCRIPTION
**Pull request by MelissaBraxton**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/help-request-template.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ---
-name: Help request 
-about: Request support from the 18F Delivery Assurance Team, each requestion should take a maximum 2 weeks.
+name: Help request
+about: Request support from the 18F Delivery Assurance Team, each requestion should
+  take a maximum 2 weeks.
```
Comment (geekygirlsarah) [inline .github/ISSUE_TEMPLATE/help-request-template.md:4]: For markdown, the `about:` front matter line can't be on two separate lines. You might need to move "take..." next to "...should"

File: .github/ISSUE_TEMPLATE/offboarding-issue-template.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+name: Offboarding issue template
+about: For ensuring a consistent offboarding experience and that nothing fall through   the
+  cracks!
```
Comment (geekygirlsarah) [inline .github/ISSUE_TEMPLATE/offboarding-issue-template.md:4]: Same here, these need to be on the same line.

File: .github/ISSUE_TEMPLATE/help-request-template.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
 ---
 name: Help request
-about: Request support from the 18F Delivery Assurance Team, each requestion should
-  take a maximum 2 weeks.
+about: Request support from the 18F Delivery Assurance Team, each requestion should take a maximum 2 weeks.
```
Comment (alexbielen) [inline .github/ISSUE_TEMPLATE/help-request-template.md:3]: Just checking on the word "requestion" -- is that on purpose?

**Review by alexbielen — CHANGES_REQUESTED** [review]
Two minor changes suggested! Otherwise looks good.

File: .github/ISSUE_TEMPLATE/help-request-template.md (hunk 3 above)
Comment (geekygirlsarah) [reply .github/ISSUE_TEMPLATE/help-request-template.md:3]: requisition?

File: .github/ISSUE_TEMPLATE/offboarding-issue-template.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+about: For ensuring a consistent offboarding experience and that nothing fall through the cracks!
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# For team leads 
+- [ ] Remove the person offboarding from 18F-TLC google group, which will uninvite them from meetings
+- [ ] Create a personalized certificate of appreciation to thank them for their contributions!
```
[the remark this answers is not in the document]
Comment (MelissaBraxton) [reply .github/ISSUE_TEMPLATE/offboarding-issue-template.md]: Made suggested revisions. @alexbielen or @kfoley-18F  - please review and merge (if all looks good), when you have a sec.

**Review by kfoley-18F — APPROVED** [review]
👏 👏 👏

**Pull request merged**